src/components/card/CardSidebarTabAttachments.vue

```diff
@@ -22,30 +22,99 @@
 
 <template>
 	<div>
-		{{ card.attachments }}
+		<li v-for="attachment in attachments" :key="attachment.id" class="attachment">
+			<a class="fileicon" :style="mimetypeForAttachment(attachment)" />
+			<div class="details" :href="attachmentUrl(attachment)">
+				<div class="filename">
+					<span class="basename">{{ attachment.data }}</span>
+				</div>
+				<span class="filesize">{{ attachment.extendedData.filesize }}</span>
+				<span class="filedate">{{ attachment.createdAt }}</span>
+				<span class="filedate">{{ attachment.createdBy }}</span>
+			</div>
+			<Actions>
+				<ActionButton icon="icon-delete" @click="deleteAttachment(attachment)">
+					{{ t('deck', 'Delete Attachment') }}
+				</ActionButton>
+
+				<ActionButton v-if="attachment.extendedData.mimetype.includes('image')" icon="icon-picture" @click="coverToggle(attachment)">
```

Comment:
SVGs should not be considered to be savely shown in a webpage if uploaded by a user. And as discussed, I'd prefer this to be in a separte PR so we can make sure we use previews instead of the full files when showing the cover.